poms/bom/pom.xml

```diff
@@ -9743,6 +9744,11 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
+                <groupId>com.github.mwiede</groupId>
+                <artifactId>jsch</artifactId>
```

Comment:
We should probably also exclude the old `com.jcraft:jsch` from `camel-quarkus-jsch`. Otherwise I think we'll end up with both the old and new jsch libs on the classpath.

If you don't want to do it here, I can follow up in another PR (it's my mistake that it wasn't done in the first instance anyway).

Reply:
Thanks!